### ISSUE TITLE
Edit Question: metadata-and-referencing/structure-granularity-and-referencing-pids/system

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#62]
**Describe the request**  
Where should the PIDs be obtained? 

**Propose what exactly should be changed**  
Perhaps question should be deleted because this is not a decision of the scientists?

**Original setting or content**   
*question text en*  
"Which system of persistent identifiers shall be used?"

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [ ] question text (en and ger)

**Comment by jmfrenzel** [comment]
Alternative question text:
If persistent identifiers are used, which type of identifiers is that.

**Comment by smohex** [comment]
UDE: I continue to ask myself whether this can actually be answered by researchers and therefore I do not find the question helpful. What answer do you expect here?
TUDO: if data is reused, there may be PIDs; I agree with Johannes' suggested wording.
UDE: Yes, there are certainly PIDs, but I am not sure whether the decision which system to use can actually be made/answered by researchers. Or if they do not take what may already be available. But yes, we can leave the question reformulated.

**Comment by jmfrenzel** [comment]
All: We leave it as is, since it's follow up question.
